src/style.css

```diff
@@ -0,0 +1,35 @@
+body {
+  font-family: Arial;
+  font-size: 13px;
+}
+
+h1 {
+  font-size: 17px;
+  font-weight: bold;
+  margin: 0;
+}
+
+h2 {
+  font-size: 15px;
+  padding-top: 1em;
+  padding-bottom: 1em;
```

Comment:
1. this is better to set as margins since it's an indentation from the nearby container
2. use short note for margin